### ISSUE TITLE
Test results are consistent with the reported values in the paper ?

### ISSUE THREAD

**Issue by LinaShanghaitech** [Engineering-Course/LIP_SSL#32]
Hi, I've attempted to test your provided best caffemodel in the validation set. 
Following the results you reported in SSL,  the overall acc., mean acc. and mIoU should be 
84.36%, 54.94%, 44.73%, respectively, but  I used the provided code  and  only got the 
82.06%, 51.64%, 40.95%.  Besides, I've also tried other methods (deeplab_v2 , attention without ssl ) according to your training strategies and all of them  get the lower results. Are there any changes to the validation set ?  Thx !
